getting_started/intel_debug/introduction.rst

```diff
@@ -160,41 +168,46 @@ Notes:
++===========================================================+========+================================================+===========+===================================+
+|Raptor Lake and older                                      |IPC3    |/lib/firmware/intel/sof/                        |sof-PLAT.ri|PLAT = glk, cml, ..., rpl          |
++-----------------------------------------------------------+        +------------------------------------------------+           |                                   |
+|Raptor Lake and older (community signed)                   |        |/lib/firmware/intel/sof/community/              |           |                                   |
++-----------------------------------------------------------+--------+------------------------------------------------+           +-----------------------------------+
+|Tiger Lake and newer                                       |IPC4    |/lib/firmware/intel/sof-ipc4/PLAT/              |           |PLAT = tgl, adl, rpl, mtl, lnl, ...|
++-----------------------------------------------------------+        +------------------------------------------------+           |                                   |
+|Tiger Lake and newer (community signed)                    |        |/lib/firmware/intel/sof-ipc4/PLAT/community/    |           |                                   |
++-----------------------------------------------------------+--------+------------------------------------------------+-----------+-----------------------------------+
+|Meteor Lake and newer Loadable libraries                   |IPC4    |/lib/firmware/intel/sof-ipc4-lib/PLAT/          |UUID.bin   |PLAT = mtl, lnl, ...               |
++-----------------------------------------------------------+        +------------------------------------------------+           |                                   |
+|Meteor Lake and newer Loadable libraries (community signed)|        |/lib/firmware/intel/sof-ipc4-lib/PLAT/community/|           |                                   |
++-----------------------------------------------------------+--------+------------------------------------------------+-----------+-----------------------------------+
+
+Look-up paths per Intel platform for **modular releases (IPC4 only)**
+
+.. _intel_modular_firmware_paths:
+
++-----------------------------------------------------------+------------------------------------------------+-------------------------+--------------------+
+|Platform                                                   |Load path                                       |File name                |Notes               |
++===========================================================+================================================+=========================+====================+
+|Meteor Lake and newer                                      |/lib/firmware/intel/sof-ipc4/PLAT/              ||                        |PLAT = mtl, lnl, ...|
```

Comment:
We already know we won't support this for MTL, LNL, so why mention these at all?

Reply:
Don't say never... Technically it is possible.